Symfony/CS/Config/Config.php

```diff
@@ -92,19 +89,7 @@ public function getFinder()
         return $this->finder;
     }
 
-    public function level($level)
-    {
-        $this->level = $level;
-
-        return $this;
-    }
-
-    public function getLevel()
-    {
-        return $this->level;
-    }
-
-    public function fixers($fixers)
+    public function fixers(array $fixers)
```

Comment:
Because on 1.x line this method can get both... names and instances... your confusion made the point.

TBD soon ;)